src/components/ReportActionItem/IOUPreview.js

```diff
@@ -133,6 +133,7 @@ const IOUPreview = (props) => {
             {style: 'currency', currency: props.iouReport.currency},
         ) : '';
     const avatarTooltip = [Str.removeSMSDomain(managerEmail), Str.removeSMSDomain(ownerEmail)];
+    const pendingConversionMessage = 'The total will be updated when connection is restored';
```

Comment:
I know this is not final, but we should probably translate this so good to move it to the en.js/ es.js already.

Reply:
Definitely.